app/src/main/res/layout/fragment_movie_details.xml

```diff
@@ -1,10 +1,272 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@color/background_color"
     tools:context=".MovieDetails">
 
+    <ImageView
+        android:id="@+id/movie_logo_image"
+        android:layout_width="match_parent"
+        android:layout_height="298dp"
```

Comment:
instead of hardcoding the heigh you can set ratio for this image

Reply:
Ok, I'll move to Aspect Ratio Constraint